kubespawner/spawner.py

```diff
@@ -674,8 +674,7 @@ def _deprecated_changed(self, change):
         config=True,
         allow_none=True,
         help="""
-        The storage class that the pvc will use. If left blank, the kubespawner will not
-        create a pvc for the pod.
+        The storage class that the pvc will use.
```

Comment:
```suggestion
        The storage class that the PVC will use. If the default value of `None`
        is used, Kubespawner will not create a PVC for the pod.
```

Reply:
I couldn't see in the code where setting storage class to `None` would prevent a pvc being created. It is valid to have `None`, meaning don't define a storage class. Kubernetes still work in that case and will use whatever the default storage class is. The comment about `None` only seemed to apply to storage size and why I removed it. It seems it was copies from other comment when storage class was added and not removed when should.

Reply:
Oh! Yes you are right, calling get_pvc_manifest will always return a manifest no matter of this, and that is a public facing function not invoked conditionally based on this.

It is probably a confusion of kubespawners use in the z2jh helm chart where such logic is implemented. Let's not commit this suggestion and resolve this conversation!